### ISSUE TITLE
Throttle and debounce options are unclear

### ISSUE THREAD

**Issue by ELLIOTTCABLE** [kimmobrunfeldt/chokidar-cli#25]
```
> chokidar --help
  -d, --debounce          Debounce timeout in ms for executing command
                                                                  [default: 400]
  -t, --throttle          Throttle timeout in ms for executing command
                                                                    [default: 0]
```

From these descriptions, it's really not clear what these *do*. For instance, I personally expected `debounce` to still *immediately execute* on the first file-change, but then ignore further changes for `n` milliseconds; instead, it waits `n` milliseconds after every change. (I then hoped that `throttle` was perhaps a badly-named tool that had the former behaviour, but that doesn't seem to be the case either!)

A little more explication in the `--help` might be worthwhile. (=

**Comment by kimmobrunfeldt** [comment]
You are right these are not explained well and could be improved. Throttle and debounce are quite standard terms so we could copy a definition somewhere. Personally I find visual explanations of them easy to grasp.
https://github.com/kimmobrunfeldt/chokidar-cli/issues/25#lock-confirmation
